sql/moz-fx-cjms-prod-f3c7/cjms_bigquery/subscriptions_v1/query.sql

```diff
@@ -0,0 +1,68 @@
+WITH aic_flows AS (
+  -- last fxa_uid for each flow_id
+  SELECT
+    flow_id,
+    MIN(flow_started) AS flow_started,
+    ARRAY_AGG(fxa_uid IGNORE NULLS ORDER BY fxa_uid_timestamp DESC LIMIT 1)[
+      SAFE_OFFSET(0)
+    ] AS fxa_uid,
+  FROM
+    `moz-fx-cjms-prod-f3c7`.cjms_bigquery.flows_v1
+  JOIN
+    EXTERNAL_QUERY("moz-fx-cjms-prod-f3c7.us.cjms-sql", "SELECT flow_id FROM aic")
+  USING
+    (flow_id)
+  WHERE
+    -- only use the last 10 days in stage
+    submission_date >= CURRENT_DATE - 10
+  GROUP BY
+    flow_id
+),
+attributed_flows AS (
+  -- last flow that started before subscription created
+  SELECT
+    stripe_subscriptions.subscription_id,
+    stripe_subscriptions.created AS subscription_created,
+    ARRAY_AGG(aic_flows.flow_id ORDER BY aic_flows.flow_started DESC LIMIT 1)[
+      SAFE_OFFSET(0)
+    ] AS flow_id,
+  FROM
+    aic_flows
+  JOIN
+    mozdata.subscription_platform.stripe_subscriptions
```

Comment:
I discussed with product, and we do not want to limit what products can be attributed during the MVP.